src/scripts/main.js

```diff
@@ -1,5 +1,27 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.addEventListener('click', (even) => {
+  const wall = even.target.closest('.wall');
+  const spider = document.querySelector('.spider');
+
+  if (!wall) {
+    return;
+  }
+
+  const wallTop = wall.offsetTop;
+  const wallLeft = wall.offsetLeft;
+
+  const clickX = even.pageX - (wallLeft + spider.offsetWidth / 2);
+  const clickY = even.pageY - (wallTop + spider.offsetHeight / 2);
+
+  const widthBorder = 20;
```

Comment:
Consider using a more descriptive name for 'widthBorder', such as 'wallBorderWidth'.